heron/packing/src/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPacking.java

```diff
@@ -284,11 +285,13 @@ private void removeInstancesFromContainers(PackingPlanBuilder packingPlanBuilder
     ArrayList<RamRequirement> ramRequirements =
         getSortedRAMInstances(componentsToScaleDown.keySet());
 
+    ContainerIdScorer scorer = new ContainerIdScorer();
```

Comment:
Is a reverse scorer for instance removal better? IMO, inserting and removing instances among the containers in reverse direct will result in better resource utilization.